src/riscv/lib/tests/expected/etherlink/result

```diff
@@ -1,5 +1,5 @@
 Exited {
-    steps: 68780951,
+    steps: 59658009,
```

Comment:
does this new Etherlink run a bit quicker? (will try locally 😄)

Reply:
indeed, goes from around *163 TPS -> 185 TPS* on my machine.

Reply:
It does! On my machine it goes from 130-140 TPS to 170-190 TPS.